Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:lts-bullseye-slim
+
+RUN mkdir -p /usr/src/teslacam-browser
```

Comment:
`WORKDIR /usr/src/teslacam-browser` will automatically[1] create the dir, can remove this line to do it manually

[1] https://docs.docker.com/engine/reference/builder/#workdir